dev.sh

```
@@ -1,69 +1,200 @@
+    
+    # Use multitail if available for best experience
+    if command -v multitail >/dev/null 2>&1; then
+        multitail \
+            -ci blue -i api_server.log \
+            -ci green -i mcp_server.log
+    else
+        # Custom log formatting with colors and timestamps
+        tail -f api_server.log mcp_server.log | while IFS= read -r line; do
+            current_time=$(date '+%H:%M:%S')
+            
+            if [[ $line =~ "==> api_server.log <==" ]]; then
```

Shellcheck (0.10.0)

[warning] Remove quotes from right-hand side of =~ to match as a regex rather than literally.

(SC2076)